torch/autograd/profiler_util.py

```diff
@@ -280,7 +281,11 @@ def export_stacks(self, path: str, metric: str):
         with open(path, "w") as f:
             for evt in self:
                 if evt.stack and len(evt.stack) > 0:
-                    metric_value = getattr(evt, metric)
+                    metric_value = getattr(evt,
```

Comment:
Could you explain why this replace is needed? thanks

Reply:
Here replacing is because the unified `self_device_time_total` is used to calculate the metrics in the `FunctionEvent`